fMRIVolume/scripts/IntensityNormalization.sh

```diff
@@ -117,20 +140,49 @@ echo "PWD = `pwd`" >> $WD/log.txt
 echo "date: `date`" >> $WD/log.txt
 echo " " >> $WD/log.txt
 
+########################################## DO WORK ##########################################
 
-########################################## DO WORK ########################################## 
-
-# FinalMask is a combination of the FS-derived brainmask, and the spatial coverage mask that captures the
-# voxels that have data available at *ALL* time points (derived in OneStepResampling)
 FinalMask=`${FSLDIR}/bin/remove_ext ${InputfMRI}`_finalmask
-${FSLDIR}/bin/fslmaths ${BrainMask} -bin -mas ${InputfMRI}_mask ${FinalMask}
+T1FOVMask=`${FSLDIR}/bin/remove_ext ${InputfMRI}`_T1FOVmask
+${FSLDIR}/bin/fslmaths ${BrainMask} -bin -mas ${InputfMRI}_mask ${T1FOVMask}
+
+# Use the requested BOLD mask 
+
+if [ "${BOLDMask}" = "T1_fMRI_FOV" ] ; then
+    # FinalMask is a combination of the FS-derived brainmask, and the spatial coverage mask that captures the
+    # voxels that have data available at *ALL* time points (derived in OneStepResampling)
+    ${FSLDIR}/bin/fslmaths ${BrainMask} -bin -mas ${InputfMRI}_mask ${FinalMask}
```

Comment:
Since this operation has already now been done above, perhaps the following here instead:
`${FSLDIR}/bin/imcp ${T1FOVMask} ${FinalMask}`

Reply:
Was contemplating the same, but thought that if we keep it separate, then it is clearer and less prone to bugs if anything changes. I've changed it to `imcp` now.